lib/internal/bootstrap/switches/is_not_main_thread.js

```diff
@@ -37,7 +37,7 @@ const {
 
 let workerStdio;
 function lazyWorkerStdio() {
-  if (!workerStdio) workerStdio = createWorkerStdio();
+  workerStdio ||= createWorkerStdio();
   return workerStdio;
```

Comment:
nit: use one line return?
```suggestion
  return workerStdio ||= createWorkerStdio();
```